src/odb/src/db/dbAccessPoint.cpp

```diff
@@ -185,6 +189,8 @@ dbOStream& operator<<(dbOStream& stream, const _dbAccessPoint& obj)
   stream << obj.bpin_;
   stream << obj.accesses_;
   stream << obj.iterms_;
+  stream << obj.vias_;
+  stream << obj.path_segs_;
```

Comment:
@Mahfouz-z move inside the user code section

Reply:
Code produced via generator now.